### ISSUE TITLE
[BUG] Site accessibility - a11y

### ISSUE THREAD

**Issue by Abhishek2634** [m2b3/SciCommons-frontend#309]
**Describe the bug**
Currenlty site is not accessible and needs a11y testing. Here i will find issues which don't follow WCAG guidelines and fix them. 

**Expected behavior**
Site should be accessible to everyone, people with different abilities.

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. macOS, Windows, Ubuntu]
 - Browser: [e.g. chrome, safari]
 - Backend Version: [e.g. 1.4.1]

**Comment by Abhishek2634** [comment]
@bsureshkrishna Sir, I have increased the scope of testing now. Now i am covering almost every browser on mobile and desktop and following every WCAG guidelines. You can check the video below. I have explain everything. Thanks.

https://github.com/user-attachments/assets/ea06d93e-155e-4918-a173-767d9a2ae423
